PgpWordList/PgpWordAggregationParser.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Messerli.PgpWordList
+{
+    public class PgpWordAggregationParser
+    {
+        public const string DefaultSeparator = "-";
+
+        private readonly string _separator;
+
+        public PgpWordAggregationParser(string separator = DefaultSeparator)
+        {
+            _separator = separator;
+        }
+
+        public byte[] Parse(PgpWordAggregation pgpWordAggregation)
+            => pgpWordAggregation
+                .Value
+                .Split(_separator)
+                .Aggregate(new List<byte>(), WordToByte)
+                .ToArray();
+
+        private static List<byte> WordToByte(List<byte> list, string word)
```

Comment:
This method does more than just convert a word to a byte, as it also appends it to a list.

Reply:
Addressed https://github.com/messerli-informatik-ag/pgp-word-list/pull/1/commits/ec053a2ee9523187cd2a046dbb6e7e605233b95e